repos/fdbt-site/tests/pages/searchOperators.test.tsx

```diff
@@ -173,7 +173,7 @@ describe('pages', () => {
                         errors: mockErrors,
                         searchText: 'asda',
```

Comment:
We probably want some tests to make sure of what happens when the user uses the page to build up the selected operators, search the repo for wrapper.find('button').simulate('click'); for an example.

Reply:
@SamerIssam  ?

Reply:
I think we just want a couple of tests that check that when you click on an operator, that it disappears from the search results and gets added to the list. And another test to ensure when you click remove, it removes a user from the list. And remove all would be another test.